fdbclient/vexillographer/fdb.options

```diff
@@ -57,6 +57,8 @@ description is not currently required but encouraged.
     <Option name="trace_file_identifier" code="36"
             paramType="String" paramDescription="The identifier that will be part of all trace file names"
             description="Once provided, this string will be used to replace the port/PID in the log file names." />
+    <Option name="trace_include_thread_ids" code="37"
```

Comment:
I think there is a compelling argument that we should just make this the default behavior. If we believe there is somebody that may not want it, then we could provide an opt-out option, or we could even choose to make it the only behavior.

@jzhou77 maybe you could comment on whether or not you all would want to use this behavior or to turn it off? The idea is to include an extra bit of info in the trace file name for clients such that we can easily distinguish between the primary network thread and all external network threads. There are also some potential implications for the configuration of the trace rolling parameters.

Reply:
Yes. I think this is a good idea to make it the default behavior. @bnamasivayam , do you think we'd ever turn this feature off?

Reply:
Thanks for the feedback. The question is then whether it is better to revert this option to disabling one or to remove it completely. One aspect to consider is that

Reply:
I now switched to using different trace file names for client threads by default, providing an option to revert the previous behavior.